src/Microsoft.TestPlatform.CommunicationUtilities/JsonDataSerializer.cs

```diff
@@ -50,6 +50,7 @@ private JsonDataSerializer()
         s_payloadSerializer = JsonSerializer.Create(jsonSettings);
         s_payloadSerializer2 = JsonSerializer.Create(jsonSettings);
 
+        var s = new DefaultTestPlatformContractResolver();
```

Comment:
Creating the resolver just once, saves time when we need both serializers.

Reply:
Could we rename the variable into something a little more explicit? (e.g. `contractResolver`, `testPlatformContractResolver`, `defaultContractResolver`).

Reply:
renamed.